spec/support/factory_bot.rb

```diff
@@ -30,24 +23,22 @@ def upload_file(file)
   #   end
   # end
 
-  # This will output total database records being created. Commented out to
-  # keep the spec output clean.
-
-  factory_bot_results = {}
-  config.before(:suite) do
-    ActiveSupport::Notifications.subscribe("factory_bot.run_factory") do |name, start, finish, id, payload|
-      factory_name = payload[:name]
-      strategy_name = payload[:strategy]
-      factory_bot_results[factory_name] ||= {}
-      factory_bot_results[factory_name][:total] ||= 0
-      factory_bot_results[factory_name][:total] += 1
-      factory_bot_results[factory_name][strategy_name] ||= 0
-      factory_bot_results[factory_name][strategy_name] += 1
+  # This will output total database records being created.
+  if ENV.fetch("SPEC_OUTPUT_FACTORY_BOT_OBJECT_CREATION_STATS", false)
```

Comment:
No need to track these results if we don't want to see them, just moved this conditional here rather than for the after(:suite) puts below.